lib/engine/step/buy_sell_par_shares.rb

```diff
@@ -30,12 +30,8 @@ def actions(entity)
       def log_pass(entity)
         return @log << "#{entity.name} passes" if @current_actions.empty?
 
-        action = if bought?
-                   'selling'
-                 else
-                   'buying'
-                 end
-        @log << "#{entity.name} passes #{action} shares"
+        action = bought? ? 'selling' : 'buying'
+        @log << "#{entity.name} declines #{action} shares"
```

Comment:
declines to buy/sell shares

Reply:
I didn't write the logic here, just changed the word, but I'm not sure it should be changed.

If you bought before hitting Done ( log would indicate "Entity declines selling shares" ), you no longer had the option to buy, and thus you declined selling ( not buy/sell )

If you sold before hitting Done ( log would indicate "Entity declines buying shares" ), then saying you declined selling (via "declines buy/sell shares") is not entirely accurate.

Reply:
Sorry, what I mean is "Entity declines selling shares" is bad English. "Entity declines to sell shares"

Reply:
Well then, that I can do!